### DISCUSSION TITLE
Coding in haxe

### DISCUSSION THREAD

**Discussion by NotStampyC** [FunkinCrew/Funkin#3076] [Q&A] [answered]
I want to attempt to make an midsong cutscene event but i literaly hwve no clue on oh to code in haxe
I can only write in python or (lua as it is similar to python) and idk what im doing

**Comment by Cartridge-Man** [comment, marked as answer]
Simply, learn the basics of how to code in Haxe by looking up what it is, downloading it, watching tutorials, testing code and stuff, it’s not too far from stuff like Java
